content/news/playtest-20160403.md

```diff
@@ -0,0 +1,118 @@
+
+Just to whet your appetite, dedicated servers no longer need to sync maps with the resource site, and it is now possible to add third-party mods as a single package (".oramod") to your installation.
+
+But one thing after the other...
+
+<hr />
+## Ingame changes
+<hr />
+
+Quite a few long-standing annoyances have been addressed during the last cycle. Just to name a few, players will no longer be kicked from the server after installing a map, walls now stop bullets reliably, and husks can be targeted on refineries.
+
+Other changes and new features include:
+
+* A couple of issues related to left-click controls have been solved.
+* Static defenses will now stop tracking their targets once it is out of range. This also sometimes prevented camo pillboxes from cloaking.
+* Canceling an entire build queue at once is now possible by using CTRL + right click.
+* Line-building of walls can now be disabled by holding the Shift key while placing the wall element.
+* Zooming is now possible by holding CTRL while scrolling your mouse wheel. For spectators and in the map editor, two additional zoom steps are available.
+* Engineers now have a lower selection priority so that they will not be selected along with your regular combat units.
+* Fragile diplomacy has been removed.
+* By popular demand, a tech level without super-powers has been added to the Red Alert mod.
+* Lots of minor fixes, UI polish and performance improvements.
+
+As usual, both the Tiberian Dawn and Red Alert mod have received some balance changes. Additionally, a number of Red Alert maps have been revised to be more balanced. A complete list of all these changes can be found [in the changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/<hash>).
+
+<div style="text-align:center" markdown="1">
+![Restart dialog](/images/news/20160403-cnc-restart-dialog.png)
+<p>
+Players finally have a restart option after failing a mission or skirmish!
+</p>
+
+![Team colors and health bar improvements](/images/news/20160403-ra-team-colors.png)
+<p>
+When team colors are enabled in the settings, they will now be used on the radar map.<br/>
+You can also set health bars to only appear when units are damaged.
+</p>
+
+![Nod units with proper color](/images/news/20160403-cnc-nod-colors.png)
+<p>
+This is for the nostalgic ones among us (and who isn't?).<br/>
+Nod units in missions now show their proper silver colors known from the original game.
+</p>
+
+![Worm trails](/images/news/20160403-d2k-worm-trail.png)
+<p>
+Worms have received some polish and now leave a trail similar to the one from the original game.
+</p>
+</div>
+
+<hr />
+## Mapping and modding changes
+<hr />
+Maps have seen many changes since the release. So many changes, in fact, that the map format number was incremented four times &ndash; from 7 to 11 &ndash; during this development cycle!
```

Comment:
Can you encorporate some of the ideas from the following?

```
The way that the game handles maps has changed significantly since the last release.  We will elaborate on the changes and the reasons for them in a future post, but for now you only really need to know three things:

<ul markdown="1">
	<li>Maps from release-20151224 and earlier must be upgraded before they will work</li>
	<li>Maps are now installed into per-version subdirectories &mdash; like replays</li>
	<li>Installing maps in-game and using the map editor works as normal</li>
</ul>

We have automatically upgraded all the maps currently available on [the OpenRA Resource Center](http://resource.openra.net), and have added an option for you to manually upgrade your maps manually.
```